### DISCUSSION TITLE
Estonian, Lithuanian, Galician, Malayalam, Telugu, Welsh new languages have been added to web-site

### DISCUSSION THREAD

**Discussion by rtsisyk** [organicmaps/organicmaps#7780] [Translations]
Six new languages have been added to https://organicmaps.app from Weblate (https://hosted.weblate.org/projects/organicmaps/). All other languages have been updated as well. Please kindly review https://organicmaps.app/ in your respective languages and make any necessary updates to https://hosted.weblate.org/projects/organicmaps/. Weblate automatically highlights issues and outdated translations that require revision.

CC: @organicmaps/translations

**Comment by jimcarst** [comment]
Translations are locked, I can only save as suggestion (nl)

**Reply by gpesquero** [reply]
~~Hi @rtsisyk: Are the FAQ of the website included in the Weblate translations? If not, how could they be included?~~

Sorry @rtsisyk: I just found the file [TRANSLATIONS.md](https://github.com/organicmaps/organicmaps.github.io/blob/master/TRANSLATIONS.md) that includes the detailed procedure to enable Weblate translations.

**Comment by ZeljkoBG** [comment]
The Serbian language has TWO letters, Cyrillic and Latin. Only Serbian is written on the weblate. It should be like this:
Serbian Latin
Serbian Cyrillic
